styles/app.css

```diff
@@ -0,0 +1,176 @@
+body {
+  /* All backgrounds to go edge-to edge */
+  margin: 0;
+  font-family: sans-serif;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+img {
+  max-width: 100%;
+}
+
+.header {
+  background-color: #4fbedb;
+  overflow: auto;
+  padding: 1rem;
+
+  /*How to curve top edges? */
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  max-width: 65rem;
+
+  margin: auto;
+
+}
+
+.header__right {
+  float: right;
+
+  margin-top: 0;
+  margin-right: 2rem;
+  padding-top:1.2rem;
+  color: white;
+
+  font-size: 1rem;
+
+}
+
+.header__span-1 {
+
+  margin-right: 3em;
+  display: inline-block;
+
+
+}
+
+.header__span-2 {
+
+  display: inline-block;
+}
+
+.header__left {
+
+  float: left;
+  display: inline-block;
+  margin-left: 2rem;
```

Comment:
Look at removing the margin-left here so that the logo lines up with the grid below